packages/yoga/src/Feedback/web/Feedback.jsx

```diff
@@ -74,7 +74,7 @@ function Feedback({ variant, title, description, children, center, ...props }) {
         />
         <TextContainer>
           {titleElement}
-          <Text.Body1 mt="small" color="deep">
+          <Text.Body1 mt="small" color="deep" as="h1">
```

Comment:
The title of this component is already an `h1`, is it expected to have two elements with level 1?

```html
<h1>
  Welcome to Yoga
</h1>
<h1>
  Enjoy your membership!
</h1>
```